criu/cgroup.c

```diff
@@ -427,10 +427,11 @@ static int dump_cg_props_array(const char *fpath, struct cgroup_dir *ncd, const
 		}
 
 		/*
-		 * Set the is_threaded flag if cgroup.type's value is threaded,
-		 * ignore all other values.
+		 * Set the is_threaded flag if cgroup.type's value is threaded or
+		 * if there is a 'task' property exists, which implies that it is a cgroup v1 (always threaded).
```

Comment:
"or it is a cgroup v1 (it has a 'task' property)."
I'd expect that cgroup v1 == always threaded is clearly implied by this shorter sentence.